inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -37,6 +37,7 @@
 )
 from inspirehep.utils.record import get_arxiv_id
 from inspirehep.utils.url import is_pdf_link
+from inspire_schemas.builders import LiteratureBuilder
```

Comment:
Nit: this guy should be at the beginning of the `inspire` imports (alphabetical order)

Reply:
changed.